src/plone/restapi/tests/test_documentation.py

```diff
@@ -785,6 +811,12 @@ def test_documentation_jwt_logout(self):
         )
         save_request_and_response_for_docs("jwt_logout", response)
 
+    def test_documentation_external_doc_links(self):
+        response = self.api_session.get(
+            f"{self.portal.absolute_url()}/@login",
+        )
+        save_request_and_response_for_docs("external_authentication_links", response)
```

Comment:
With my rename of the docs file, should this be changed as well?

```suggestion
        save_request_and_response_for_docs("login", response)
```

Reply:
No, this is for the test-generated request and response file. I would leave this as it is, because we already have `login.req` file in the http-examples folder